tests/integration_tests/data_steward/utils/bq_test.py

```diff
@@ -60,3 +74,45 @@ def test_define_dataset(self):
         # Post conditions
         self.assertIsInstance(results, bigquery.Dataset)
         self.assertEqual(results.labels, self.label_or_tag)
+
+    def test_update_labels_and_tags(self):
+        # Tests if project_id is given
+        self.assertRaises(TypeError, update_labels_and_tags, self.dataset_id,
+                          self.label_or_tag)
+
+        # Tests if dataset_id is given
+        self.assertRaises(TypeError, update_labels_and_tags, self.project_id,
+                          self.label_or_tag)
+
+        # Tests if no label or tag is given
+        self.assertRaises(TypeError, update_labels_and_tags, self.project_id,
+                          self.dataset_id)
+
+        # Pre-conditions
+        results = define_dataset(self.project_id, self.dataset_id,
+                                 self.description, self.label_or_tag)
+
+        self.client = bigquery.Client()
+        self.dataset = self.client.create_dataset(results)
```

Comment:
This may be debatable, but I would argue this should be implemented not as an integration test, but as a regular unit test. In a unit test we would mock API calls and doing so would allow us to forgo actual dataset creation/deletion. As is, it seems to me that multiple clients may disrupt one another by running this test suite at the same time. For example this sequence of events seems possible:
1. Mark executes this line creating `foo_dataset`
1. Danyel executes `tearDown()` deleting `foo_dataset`
1. Mark executes the following lines which would fail because the dataset disappeared
CC: @lrwb-aou